systori/apps/company/middleware.py

```diff
@@ -24,6 +24,21 @@ def process_request(request):
 
         request.company = None
 
+        is_android_development = (
```

Comment:
I think you should wrap this with a check if django is running in a dev environment...

something like
```python
from django.conf import settings

if settings.DEBUG:
    ....
```